.github/workflows/coverage.yml

```diff
@@ -0,0 +1,28 @@
+name: coverage
+on:
+  push:
+    branches:
+      - 'main'
+      - 'scratch/**'
+  pull_request:
+
+jobs:
+  coverage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: metrumresearchgroup/actions/mpn-latest@v1
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          r-version: release
+          use-public-rspm: true
+          extra-repositories: 'https://mpn.metworx.com/snapshots/stable/${{ env.MPN_LATEST }}'
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          extra-packages: any::covr
+          upgrade: 'TRUE'
+      - name: Run covr
+        shell: Rscript {0}
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+        run: covr::codecov()
```

Comment:
`covr::codecov` isn't running successfully.

https://github.com/metrumresearchgroup/pmforest/actions/runs/10289387264/job/28477016357

```
2024-08-07T18:06:58.4091002Z ##[group]Run covr::codecov()
2024-08-07T18:06:58.4091336Z [36;1mcovr::codecov()[0m
2024-08-07T18:06:58.4097627Z shell: /usr/local/bin/Rscript {0}
2024-08-07T18:06:58.4097930Z env:
2024-08-07T18:06:58.4098145Z   MPN_LATEST: 2024-06-12
2024-08-07T18:06:58.4098453Z   R_LIBS_USER: /home/runner/work/_temp/Library
2024-08-07T18:06:58.4098784Z   TZ: UTC
2024-08-07T18:06:58.4099008Z   _R_CHECK_SYSTEM_CLOCK_: FALSE
2024-08-07T18:06:58.4099287Z   NOT_CRAN: true
2024-08-07T18:06:58.4099667Z   RSPM: https://packagemanager.posit.co/cran/__linux__/jammy/latest
2024-08-07T18:06:58.4100317Z   RENV_CONFIG_REPOS_OVERRIDE: https://packagemanager.posit.co/cran/__linux__/jammy/latest
2024-08-07T18:06:58.4100873Z   R_LIB_FOR_PAK: /opt/R/4.4.1/lib/R/site-library
2024-08-07T18:06:58.4101234Z   CODECOV_TOKEN: 
2024-08-07T18:06:58.4101474Z ##[endgroup]
2024-08-07T18:07:14.0761631Z Request failed [429]. Retrying in 1 seconds...
2024-08-07T18:07:15.2792610Z Request failed [429]. Retrying in 3.7 seconds...
2024-08-07T18:07:19.2459084Z $detail
2024-08-07T18:07:19.2460711Z [1] "Rate limit reached. Please upload with the Codecov repository upload token to resolve issue. Expected time to availability: 1924s."
```

Notice that `CODECOV_TOKEN` is empty.  (It'd be marked by asterisks if it were set by a secret.  See bbr's coverage action output.)

So you should either configure that or drop this.  (I think the latter is fine.)

The b57855d message says "coverage display wasn't set up before, so leaving that alone in the readme for now".  I don't follow that, since I don't think the switch to actions would require adjusting the URL.  However, if you don't end up configure this workflow, in my opinion it'd be good to drop the badge because there's no point in leaving a non-functional link in the README.

Reply:
Yeah assumed some additional setup was required to get that working. Im not familiar with the steps and many of our other packages dont run this, so opting to just remove.